app/src/main/java/org/fossasia/openevent/app/Views/EventsActivity.java

```diff
@@ -0,0 +1,78 @@
+package org.fossasia.openevent.app.Views;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.android.volley.VolleyError;
+import com.google.gson.Gson;
+
+import org.fossasia.openevent.app.Adapters.EventListAdapter;
+import org.fossasia.openevent.app.Api.ApiCall;
+import org.fossasia.openevent.app.Interfaces.VolleyCallBack;
+import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.Utils.Constants;
+import org.fossasia.openevent.app.Utils.Network;
+import org.fossasia.openevent.app.model.UserEvents;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class EventsActivity extends AppCompatActivity {
+
+    public static final String TAG = "EventsActivity";
+    RecyclerView recyclerView;
+    static ArrayList<UserEvents> userEventsArrayList = new ArrayList<>();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_events);
+        recyclerView = (RecyclerView) findViewById(R.id.rvEventList);
+        RecyclerView.LayoutManager layoutManager = new LinearLayoutManager(this);
+        recyclerView.setLayoutManager(layoutManager);
+        final EventListAdapter eventListAdapter = new EventListAdapter(userEventsArrayList , this);
+        recyclerView.setAdapter(eventListAdapter);
+        RecyclerView.ItemDecoration itemDecoration = new DividerItemDecoration(this , DividerItemDecoration.VERTICAL);
+        recyclerView.addItemDecoration(itemDecoration);
+
+
+        VolleyCallBack volleyCallBack = new VolleyCallBack() {
+            @Override
+            public void onSuccess(String response) {
+                Gson gson = new Gson();
+                UserEvents[] userEvents = gson.fromJson(response , UserEvents[].class);
+                userEvents = Arrays.copyOfRange(userEvents,1,userEvents.length);
```

Comment:
userEvents = Arrays.copyOfRange(userEvents,1,userEvents.length); 

is giving an error if userEvents.length is 0 and the app is crashing